notifier/build.gradle.kts

```diff
@@ -20,6 +20,14 @@
 plugins {
     // Apply precompiled plugins.
     id("ort-library-conventions")
+    id("com.netflix.nebula.jakartaee-migration") version "0.24.0"
+}
+
+jakartaeeMigration {
+    includeTransform("com.atlassian.jira:jira-rest-java-client-core")
+    configurations.filter { "detekt" !in it.name }.forEach { config ->
```

Comment:
@sschuberth I am not happy with this `filter` here. Maybe there is a better way to determine the configurations in which this plugin should be applied.

Reply:
Does `preventTransformsOfProductionConfigurations()` work instead?

Reply:
I've tried
```
    preventTransformsOfProductionConfigurations()
    configurations.forEach { config ->
        transform(config)
    }
``` 

But then it complains:
`Use of transforms on production configurations is not allowed (configuration: api)`

So we would have to specify the configurations by hand anyway.

Also I don't get it: we _do want_ the plugin to be active on production code, i.e. when producing a library for an ORT module ?

Reply:
This seems to work more generally (without `preventTransformsOfProductionConfigurations()`:

    configurations.filterNot { it.isCanBeDeclared }.forEach(::transform)